plaidml/edsl/tests/edsl_test.cc

```diff
@@ -877,6 +877,17 @@ TEST_F(CppEdsl, Reciprocal) {
   checkProgram(program, {{A, input}}, {{R, expected}});
 }
 
+TEST_F(CppEdsl, ReshapeFold) {
+  auto A = Placeholder(DType::INT32, {3, 3});
+  auto R = reshape(A, {3, 3});
+  auto program = makeProgram("reshape_fold", {R});
```

Comment:
`program` is unused. Also, can you call `checkProgram()` so that it actually runs?